src/pages/conversion/style.js

```diff
@@ -3,7 +3,7 @@ import { css, keyframes } from '@emotion/css';
 import { color, fontSize, fontWeight } from 'common/styles';
 
 export const container = css`
-  position: fixed;
+  // position: fixed;
```

Comment:
Why commented out this?

Reply:
Both this and the next one were causing issue, as they were hiding the banner because position=fixed. Hence, had to remove that. It works fine after as well, but the banner is now visible on top.